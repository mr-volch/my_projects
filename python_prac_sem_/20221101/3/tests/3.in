f, g, h = Grange(2, 2, 12), Grange(5, 5, 15625), Grange(1, 10, 100000)
print(f"{f=}, {g=}, {h=}")
print(f"{len(f)=}, {len(g)=}, {len(h)=}")
print(f"{len(list(f))=}, {len(tuple(g))=}, {len(list(h))=}")
print(f"{f[421]=}, {h[244]=}")
a, b = f[5:532525], h[1:53252:9]
print(f"{a=}, {a[2]=}")
print(f"{b=}, {b[2]=}, {list(b)=}")
o = Grange(1, 800, 799)
print(list(o), not(not(o)))
print(o[3])
