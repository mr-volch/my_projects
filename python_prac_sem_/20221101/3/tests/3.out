f=grange(2, 2, 12), g=grange(5, 5, 15625), h=grange(1, 10, 100000)
len(f)=3, len(g)=5, len(h)=5
len(list(f))=3, len(tuple(g))=5, len(list(h))=5
f[421]=10830740992659433045228180406808920716548582325686783496759685861775864483615725089999900023844295226942934417817982702456930304, h[244]=10000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000
a=grange(5, 2, 532525), a[2]=20
b=grange(1, 1000000000, 53252), b[2]=1000000000000000000, list(b)=[1]
[1] True
512000000
